pkg/cmd/server/origin/master.go

```diff
@@ -492,6 +496,27 @@ func (c *MasterConfig) GetRestStorage() map[string]rest.Storage {
 		storage["builds/log"] = buildlogregistry.NewREST(buildStorage, buildStorage, c.BuildLogClient(), kubeletClient)
 	}
 
+	if sdnfactory.IsMultitenantNetworkPlugin(c.Options.NetworkConfig.NetworkPluginName) {
+		netNamespaceStorage := netnamespaceetcd.NewREST(c.EtcdHelper)
+		netNamespaceRegistry := netnamespace.NewRegistry(netNamespaceStorage)
+		c.MultitenantNetworkConfig.NetNamespaceRegistry = netNamespaceRegistry
```

Comment:
We shouldn't mutate the config inside this function. What's a better way to get the `MultitenantNetworkConfig` set up